deployment/helm/synopsys-alert/templates/alert-environ-configmap.yaml

```diff
@@ -1,13 +1,20 @@
 apiVersion: v1
 kind: ConfigMap
 data:
+  {{- if .Values.alert.hostname }}
+  ALERT_HOSTNAME: {{ .Values.alert.hostname | quote }}
+  {{- else }}
+  ALERT_HOSTNAME: {{ .Release.Name | quote }}
```

Comment:
Since the default value is provided as localhost, it will always go to if condition. And also, if someone want to set a different hostname, then they can set them in the flag. So ideally, we can remove the if-else condition and always read it from .Values.alert.hostname